setup/install/providers/azure.md

```diff
@@ -79,11 +85,19 @@ hal config provider azure account add my-azure-account \
   --subscription-id $SUBSCRIPTION_ID \
   --default-key-vault $VAULT_NAME \
   --default-resource-group $RESOURCE_GROUP \
+  --packer-resource-group $RESOURCE_GROUP \
   --app-key
 ```
 
-> NOTE: You will be prompted for the App Key on standard input. If necessary,
-you can generate a new key: `az ad sp credential reset --name $APP_ID`
+> NOTE:
+> 1. You will be prompted for the App Key on standard input. If necessary,
+you can generate a new key: `az ad sp credential reset --name $APP_ID`<br/>
```

Comment:
Prefer using Markdown blank lines instead of HTML line breaks.

Reply:
Fixed